src/HUDL/HUDL.cpp

```diff
@@ -31,35 +31,58 @@ CO_OBJ_T* HUDL::getObjectDictionary() {
     return &objectDictionary[0];
 }
 
-uint16_t HUDL::getObjectDictionarySize() const {
+uint16_t HUDL::getObjectDictionarySize() {
     return OBJECT_DICTIONARY_SIZE;
 }
 
 void HUDL::updateLCD() {
+    //    lcd.writeText("Hello World", 1, 0, false);
     char tempOne[32];
-    std::sprintf(tempOne, "%lu C", *(this->thermTemps + 0));
+    std::sprintf(tempOne, "%d C", *(this->thermTemps + 0));
 
     char tempTwo[32];
```

Comment:
The temps would be a little nicer in a 2-D array instead of 4 separate arrays. It would be something like  `char temps[32][4];` If you do it this way, you could convert these first four `sprintf`s into a for loop, which would be much cleaner